python/cmake_modules/FindParquet.cmake

```diff
@@ -0,0 +1,80 @@
+# Copyright 2012 Cloudera Inc.
```

Comment:
Is this necessary? The ../cpp/cmake_module is also being included in the CMAKE_MODULE_PATH, or that was the idea